File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]: 
        k=[]
        for i in range (len(nums)):
            for j in range(i+1,len(nums)):
                if target-nums[i]==nums[j]:
                    k=[i,j]
                    
        return k
        

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> map = new HashMap<>();
        map.put(nums[0],0);
        int num1=-1;
        int num2=-1;
        for(int i=1;i<nums.length;i++){
            int rem  = target-nums[i];
            if(map.containsKey(rem)){
                num1=map.get(rem);
                num2=i;
                break;
            } 
            map.put(nums[i],i);
        }
        return new int[]{num1,num2};
    }
}